foodWise/src/main/java/br/com/foodWise/foodWise/service/exceptions/user/UserNotFound.java

```diff
@@ -0,0 +1,10 @@
+package br.com.foodWise.foodWise.service.exceptions.user;
+
+import br.com.foodWise.foodWise.controllers.handlers.BusinessException;
+import org.springframework.http.HttpStatus;
+
+public class UserNotFound extends BusinessException {
+    public UserNotFound() {
```

Comment:
Acredito que podemos mudar para Resource Not Found pois teremos essa exception para produtos, restaurantes, customers, etc, então usariamos um arquivo só e podemos customizar apenas a mensagem concatenando 'X' não encontrado